setup/db/db/schema-480to500-cleanup.sql

```diff
@@ -3,10 +3,10 @@
 --;
 
 # Remove NetApp plugin
+DROP TABLE IF EXISTS `cloud`.`netapp_lun`;
 DROP TABLE IF EXISTS `cloud`.`netapp_volume`;
 DROP TABLE IF EXISTS `cloud`.`netapp_pool`;
-DROP TABLE IF EXISTS `cloud`.`netapp_lun`;
 
 # Remove BigSwitch plugin
+DROP TABLE IF EXISTS `cloud`.`external_bigswitch_vns_devices`;
 DROP TABLE IF EXISTS `cloud`.`external_bigswitch_bcf_devices`;
```

Comment:
@borisroman FYI: There is also a table called 'cloud.brocade_network_vlan_map', at least in NL2.